pyalgotrade/optimizer/xmlrpcserver.py

```diff
@@ -89,6 +87,9 @@ def __init__(self, paramSource, resultSinc, barFeed, address, port, autoStop=Tru
         self.register_function(self.getNextJob, 'getNextJob')
         self.register_function(self.pushJobResults, 'pushJobResults')
 
+    def getBatchSize(self):
```

Comment:
Do you really need this method ? Unless you're doing something custom access to the Server instance is not possible, and PyAlgoTrade is not using it, so it affects code coverage.